app/assets/stylesheets/global.scss

```diff
@@ -115,3 +115,21 @@ textarea {
   font-family: $body-fonts;
   padding: 0.7rem 1rem 0.7rem 1rem;
 }
+
+a.internal-link {
+  color: $text-primary;
+  text-decoration: none;
+  padding-left: 1.1em;
+  margin-left: -1.1em;
```

Comment:
why do we need a minus margin here?

Reply:
that's a trick to get the background image to appear to the right